### DISCUSSION TITLE
Rule proposal: @Input() decorator in an EventEmitter()

### DISCUSSION THREAD

**Discussion by FonecaBot2** [angular-eslint/angular-eslint#1170] [New Rule Proposals]
This is something that I saw on the Angular project (https://github.com/angular/angular/issues/47420). As stated by one of the maintainers there, a lint check probably would work. I would like to try this if it's accepted.

**Comment by JamesHenry** [comment]
We will gladly accept this rule thank you, please kindly respond to the existing PR feedback you have on your open PR
